src/coprocessor/executor/aggregation.rs

```diff
@@ -35,27 +35,23 @@ pub struct AggregationExecutor<'a> {
     cursor: usize,
     executed: bool,
     ctx: Rc<EvalContext>,
-    cols: Vec<ColumnInfo>,
+    cols: Rc<Vec<ColumnInfo>>,
+    related_cols: Vec<usize>, // offset of related columns
```

Comment:
Better to use name consistently.

| name | represent |
| ---- | ---- |
| column/column_info/info | Column Info |
| id/column_id | Column ID |
| offset | Column Offset |